test.mk

```diff
@@ -11,6 +11,10 @@ ovmf-aa64:
 	mkdir -p ovmf-aa64
 	cd ovmf-aa64 && curl -o OVMF-AA64.zip https://efi.akeo.ie/OVMF/OVMF-AA64.zip && 7z x OVMF-AA64.zip
 
+ovmf-rv64:
+	mkdir -p ovmf-rv64
+	cd ovmf-rv64 && curl -o OVMF.fd https://retrage.github.io/edk2-nightly/bin/RELEASERISCV64_VIRT.fd
```

Comment:
Should this be expanding the firmware file to 33554432 bytes or whatever? I guess that may be a moving target depending on the QEMU version...